### PR TITLE
[FIX] sql_export_mail : do not raise an error if non admin user want to access to sql export

### PR DESCRIPTION
**Pull request by legalsylvain**
Alternative implementation of https://github.com/OCA/reporting-engine/pull/771

### CODE REVIEW

File: sql_export_mail/views/sql_export_view.xml [hunk 1]
```diff
@@ -36,6 +37,7 @@
                             nolabel="1"
                             colspan="2"
                             domain="[('model_id', '=', 'sql.export')]"
+                            groups="base.group_system"
```
Comment (florian-dacosta) [inline sql_export_mail/views/sql_export_view.xml]: I would put the cron on the group level (or both). Else, the user  without group will just see the group name with nothing else and it is kind of strange.
Apart from this, (and pre-commit small diff)  LGTM, thanks

**Comment by legalsylvain** [comment]
Thanks ! 
done.

**Review by florian-dacosta — APPROVED** [review]

**Review by bguillot — APPROVED** [review]

**Comment by bguillot** [comment]
Thanks !

**Pull request merged**